frontend/src/features/SideWindow/MissionForm/ActionForm/LandControlForm.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  Checkbox,
+  FormikCheckbox,
+  FormikDatePicker,
+  FormikEffect,
+  FormikTextarea,
+  FormikTextInput,
+  Icon
+} from '@mtes-mct/monitor-ui'
+import { Formik } from 'formik'
+import { noop } from 'lodash'
+import { useMemo } from 'react'
+
+import { ControlQualityField } from './shared/ControlQualityField'
+import { FleetSegmentsField } from './shared/FleetSegmentsField'
+import { FormikMultiInfractionPicker } from './shared/FormikMultiInfractionPicker'
```

Comment:
Some formik fields are called `[...]Field` and some `Formik[...]`, no ?

Reply:
Yes because I want to distinguish reusable components (with the goal of sharing then in Monitor UI) from final ones but we can indeed name everything Formik.